src/graphiql-explain/constants/thresholds.jsx

```diff
@@ -0,0 +1,42 @@
+import { NegativeIndicator } from '../../icons/negative-indicator'
+import { PositiveIndicator } from '../../icons/positive-indicator'
+import { NeutralIndicator } from '../../icons/neutral-indicator'
+
+const lightColors = {
+  low: '#21835F',
+  medium: '#9E5F00',
+  high: '#D43408',
+  default: 'inherit'
+}
+
+const darkColors = {
+  low: '#00D688',
+  medium: '#FFCC99',
+  high: '#FF5729',
+  default: 'inherit'
+}
+
+export const isLight = () =>
+  !!document.querySelectorAll('.graphiql-light').length
+
+export const colors = isLight() ? lightColors : darkColors
```

Comment:
let's say strictly speaking this wouldn't need to be a function (I mean isLight)